static/script.js

```diff
@@ -1,11 +1,32 @@
 $(function () {
+  window.setInterval(refresh, 30000)
+
   $('[data-localtime="true"]').each(function () {
     let $element = $(this)
     let dateTime = moment.utc($element.text().trim())
     dateTime.local()
     $element.text(dateTime.format('YYYY-MM-DD HH:mm:ss'))
   })
+
   $('.no-collapsable').on('click', function (event) {
     event.stopImmediatePropagation()
   })
+  $('#auto-refresh').on('change', function (event) {
+    if (localStorage.getItem('refresh_f')) {
+      localStorage.removeItem('refresh_f')
+    } else {
+      localStorage.setItem('refresh_f', 'true')
+    }
+  })
+
+  function refresh () {
+    if (localStorage.getItem('refresh_f')) {
+      window.location.reload()
+      localStorage.setItem('refresh_f', 'true')
```

Comment:
This is not necessary here. Refresh function should have only single purpose: To refresh website.
Only place which touch local storage should be `$('#auto-refresh').on('change', function (event) {})`.